2022-04-05 13:24:23,019 INFO    MainThread:29364 [wandb_setup.py:_flush():71] setting env: {}
2022-04-05 13:24:23,019 INFO    MainThread:29364 [wandb_setup.py:_flush():71] setting login settings: {}
2022-04-05 13:24:23,020 INFO    MainThread:29364 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\hplis\OneDrive\Documents\GitHub\vaderSentiment\wandb\run-20220405_132423-1iwdl86d\logs\debug.log
2022-04-05 13:24:23,020 INFO    MainThread:29364 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\hplis\OneDrive\Documents\GitHub\vaderSentiment\wandb\run-20220405_132423-1iwdl86d\logs\debug-internal.log
2022-04-05 13:24:23,020 INFO    MainThread:29364 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000023E85635700>
2022-04-05 13:24:23,020 INFO    MainThread:29364 [wandb_init.py:init():404] calling init triggers
2022-04-05 13:24:23,020 INFO    MainThread:29364 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-04-05 13:24:23,020 INFO    MainThread:29364 [wandb_init.py:init():460] starting backend
2022-04-05 13:24:23,020 INFO    MainThread:29364 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-04-05 13:24:23,025 INFO    MainThread:29364 [backend.py:ensure_launched():216] starting backend process...
2022-04-05 13:24:23,090 INFO    MainThread:29364 [backend.py:ensure_launched():221] started backend process with pid: 28636
2022-04-05 13:24:23,091 INFO    MainThread:29364 [wandb_init.py:init():469] backend started and connected
2022-04-05 13:24:23,099 INFO    MainThread:29364 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-04-05 13:24:23,100 INFO    MainThread:29364 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-04-05 13:24:23,100 INFO    MainThread:29364 [wandb_init.py:init():533] updated telemetry
2022-04-05 13:24:23,165 INFO    MainThread:29364 [wandb_init.py:init():563] communicating current version
2022-04-05 13:24:24,450 INFO    MainThread:29364 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-05 13:24:24,450 INFO    MainThread:29364 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-04-05 13:24:24,921 INFO    MainThread:29364 [wandb_init.py:init():606] starting run threads in backend
2022-04-05 13:24:29,930 INFO    MainThread:29364 [wandb_run.py:_console_start():1810] atexit reg
2022-04-05 13:24:29,930 INFO    MainThread:29364 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-04-05 13:24:29,930 INFO    MainThread:29364 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-04-05 13:24:29,932 INFO    MainThread:29364 [wandb_run.py:_redirect():1745] Redirects installed.
2022-04-05 13:24:29,933 INFO    MainThread:29364 [wandb_init.py:init():633] run started, returning control to user process
2022-04-05 13:24:29,933 INFO    MainThread:29364 [wandb_watch.py:watch():43] Watching
2022-04-05 13:25:08,002 INFO    MainThread:29364 [wandb_init.py:_pause_backend():284] pausing backend
2022-04-05 13:25:08,002 INFO    MainThread:29364 [jupyter.py:save_ipynb():374] not saving jupyter notebook
